### DISCUSSION TITLE
Increments in week of the day not working if all days of week are selected

### DISCUSSION THREAD

**Discussion by ajaykumarbommena** [quartznet/quartznet#974] [General] [answered]
Example
0 0 12 ? * SUN,MON,TUE,WED,THU,FRI,SAT/2

Should this run all day of the week but on alternate weeks?

**Comment by lahma** [comment, marked as answer]
I think you shouldn't try to achieve this with just a cron trigger. You can use calendars to further restrict the firing. So basically you can have `ICalendar` implementation with `IsTimeIncluded` that checks if week is odd/even.

**Comment by ajaykumarbommena** [comment]
Ok. Sure. Thank you
